src/templates/press-page.js

```diff
@@ -31,9 +31,7 @@ const PressItem = ({ title, date, source, aboveTheFold, url }) => {
               buttons={false}
             />
           </div>
-          <div className="view-arrow">
-            View <ArrowNorthEast />
-          </div>
+          <div className="view-arrow">View &#x2197;</div>
```

Comment:
Fwiw, my preference on these when possible is to just insert them as Unicode in the first place. It helps the html show more clearly what will ultimately be on the page. Since all our files should be UTF-8, that should Just Work™, though I don't know if Gatsby will mangle something (I'm assuming not).

Reply:
I think it works fine just as Unicode (based on past uses in Gatsby) - will update!